src/Umbraco.Configuration/Models/MachineKeyConfig.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Microsoft.Extensions.Configuration;
+using Umbraco.Core.Configuration;
+
+namespace Umbraco.Configuration.Models
+{
+    public class MachineKeyConfig : IMachineKeyConfig
+    {
+        private readonly IConfiguration _configuration;
+        public MachineKeyConfig(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        //TODO all the machineKey stuff should be replaced: https://docs.microsoft.com/en-us/aspnet/core/security/data-protection/compatibility/replacing-machinekey?view=aspnetcore-3.1
```

Comment:
New GitHub issue to open if not already?

Reply:
We should remove Machine Key config entirely, there is no need for this at all in aspnetcore for our needs.